### ISSUE TITLE
Unable to connect to meganserver from Megan6: Error: Execute failed: java.io.IOException: fixed content-length

### ISSUE THREAD

**Issue by jpetucci** [husonlab/megan-ce#11]
I am using the latest available version of Megan6 and meganserver (v6.20.11) on machines running RHEL v7.8. For the purposes of simplification, I am attempting to connect to meganserver from Megan6 on the same machine (the same issue arises if meganserver runs on a separate machine).

**meganserver startup and output:**
```bash
[jmp579@comp-lc-0016 tools]$ bash megan-server -i /gpfs/group/LiberalArts/default/lsw132_collab/IntersectTransfer_20200302/MEGAN_SERVER -g -v -d
MeganServer - Serves MEGAN files over the web via HTTP
Options:
Input
        --input: /gpfs/group/LiberalArts/default/lsw132_collab/IntersectTransfer_20200302/MEGAN_SERVER
        --recurse: true
        --extensions: .daa .rma .rma6 .megan .megan.gz
Server
        --endpoint: megan6server
        --port: 8001
        --allowGuest: true
Other:
        --usersFile: /storage/home/j/jmp579/.MeganServerUsers.def
        --maxBacklog: 100
        --pageTimeout: 10000
        --debug: true
        --verbose: true
Version   MEGAN Community Edition (version 6.20.11, built 2 Nov 2020)
Author(s) Daniel H. Huson
Copyright (C) 2020 Daniel H. Huson. This program comes with ABSOLUTELY NO WARRANTY.
Updating file: /storage/home/j/jmp579/.MeganServerUsers.def
Guests can login with name: guest and password: guest
Starting http server...
MeganServer
Version: MEGAN Community Edition (version 6.20.11, built 2 Nov 2020)
Hostname: comp-lc-0016.acib.production.int.aci.ics.psu.edu
IP address: 10.102.151.16
Port: 8001
Known users: 3
Total requests: 1
Server started: Wed Nov 04 09:27:12 EST 2020
Help: http://10.102.151.16:8001/megan6server/help

Server address:
http://10.102.151.16:8001megan6server
http://comp-lc-0016.acib.production.int.aci.ics.psu.edu:8001megan6server

Rebuilding database:
10% 20% 30% 40% 50% 60% 70% 80% 90% 100% (296.2s)
Files: 4,664
```
**Note that there is a "/" missing above in the reported server address.**

With Megan6, I tried connecting to the running instance of meganserver at the server address of: "http://10.102.151.16:8001/megan6server" with both a guest account and another account which was created using the web admin commands. The output from megan6 is:
```bash
Version   MEGAN Community Edition (version 6.20.11, built 2 Nov 2020)
Author(s) Daniel H. Huson
Copyright (C) 2020 Daniel H. Huson. This program comes with ABSOLUTELY NO WARRANTY.
This is free software, licensed under the terms of the GNU General Public License, Version 3.
Sources available at: https://github.com/husonlab/megan-ce
Java version: 14.0.1

Opening startup files
Loading ncbi.map: 2,259,889
Loading ncbi.tre: 2,259,893
Executing: show window=RemoteBrowser;
Executing: openServer url='http://10.102.151.16:8001/megan6server' user='Megan' password='******';
IOException: fixed content-length: 1, bytes received: 0
Command usage: openServer url=<url> [user=<user>] [password=<hiddenPassword>]; - Open a remote server
IOException: fixed content-length: 1, bytes received: 0
Error: Execute failed: java.io.IOException: fixed content-length: 1, bytes received: 0
Executing: openServer url='http://10.102.151.16:8001/megan6server' user='guest' password='******';
IOException: fixed content-length: 6, bytes received: 0
Command usage: openServer url=<url> [user=<user>] [password=<hiddenPassword>]; - Open a remote server
IOException: fixed content-length: 6, bytes received: 0
Error: Execute failed: java.io.IOException: fixed content-length: 6, bytes received: 0
```

Using a browser on the machine running meganserver, I get a 404 error when navigating to "http://10.102.151.16:8001/megan6server", however other pages such as "http://10.102.151.16:8001/megan6server/about" work OK after signing in.

One oddity is that I am able to connect to meganserver when it is initially started up and before the "Rebuilding database" step completes. 0 files are reported during this time and when the rebuilding finishes the state cannot be refreshed and I cannot reconnect.

**Comment by husonlab** [comment]
Thanks for catching the missing "/" in the URL output. I will fix this.

Encouraging to hear that you can interact with MeganServer through a web browser.
So does http://10.102.151.16:8001/megan6server/list  work?
What does http://10.102.151.16:8001/megan6server/about report?

Are you running MEGAN on the same machine as MeganServer?

I will upload a new release  6.20.12 with three changes:
1) I have fixed the missing slash issue
2) If you run the MEGAN with command line option -d then you will see all the requests that MEGAN makes. To help me debug, could you please copy and paste some of them into your web-browser to see whether they work there.
Then we would know whether MEGAN is making the right requests, but is failing due to permissions or something.
3) I have also modified the code so that it doesn't choke on some megan files that are missing certain classifications.

**Comment by jpetucci** [comment]
> So does http://10.102.151.16:8001/megan6server/list work?

Yes, this shows the following through the web browser
```bash
2014nr-LCAdefault/00_OTHER_SAMPLES/150223_ELSIDRON_collapsed.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/150526_Microbiome_SIM80_1_MALT.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_EBC_13344.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_HG_12014.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_HG_12017.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_IR_13232.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_IR_13234.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_IR_13236.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_JEWB_8812.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_JEWB_8824.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_LBK_12824.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_LBK_12826.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AFR_transect_rma/AFR_LBK_12829.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/AIR_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/BRACCISHWATER_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/DC_CHIMP_ColR1R2_CatL7L8_150520_MALT.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/EBC_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/ELSIDRON2_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/FRESHWATER_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/GRASSLANDSOIL_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/INDREV_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/MODERN_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/MicrobiomeSIM120_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/MicrobiomeSIM40_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/SOILUSA_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/SPYNEW_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/SPYOLD_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/Warinner_B61_MALT_ALL.rma
2014nr-LCAdefault/00_OTHER_SAMPLES/Warinner_G12_MALT_ALL.rma
2014nr-LCAdefault/AGF-X10-20180122/Hinxton16489-X10.rma6
2014nr-LCAdefault/AGF-X10-20180122/Hinxton16490-X10.rma6
2014nr-LCAdefault/AGF-X10-20180122/Raunds8869-X10.rma6
2014nr-LCAdefault/AGF-X10-20180122/Raunds8872-X10.rma6
2014nr-LCAdefault/AGF-X10-20180122/StHelensOnWalls8873-X10.rma6
2014nr-LCAdefault/AGF-X10-20180122/StHelensOnWalls8874-X10.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/12893_Monongahela_Early_Reextracted__R1TAGGCCA_R2GCACTTG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/12897_Monongahela_Early_Reextracted__R1AGATTCC_R2TGCGAAT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/12898_Monongahela_Early_Reextracted__R1CTCAAGG_R2ATGTCCA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/12901_Monongahela_Middle_Reextracted__R1GAGCCTT_R2GTTGACT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/12902_Monongahela_Middle_Reextracted__R1TCTGGAA_R2TAATCGA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/12904_Monongahela_Middle_Reextracted__R1CTAGACA_R2ACCAGTC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/12916_Monongahela_Late_Reextracted__R1GACTCGC_R2CGGCTAG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19767_ModernCalculus_PerioDisease_1__R1CGCTGAG_R2AAGCATC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19768_ModernCalculus_PerioDisease_2__R1TCATGAT_R2GACGAGT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19769_ModernCalculus_PerioDisease_3__R1CAGGAAT_R2TCGTCTA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19770_ModernCalculus_PerioDisease_4__R1GCTTCCA_R2AGTAGAC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19771_ModernCalculus_PerioDisease_5__R1GTGATCT_R2CTAACAG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19772_ModernCalculus_PerioDisease_6__R1TGAAGGC_R2GGAGTAC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19773_ModernCalculus_PerioDisease_7__R1ATCCTTG_R2TTCTACG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19774_ModernCalculus_PerioDisease_8__R1ATCGATT_R2CCTCGTA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19775_ModernCalculus_PerioDisease_9__R1CAGTCAA_R2ACTTGGC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19776_ModernCalculus_PerioDisease_10__R1TATCAGA_R2GACCGCT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19777_ModernCalculus_PerioDisease_11__R1GCTAGCC_R2CGAATTG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19778_ModernCalculus_PerioDisease_12__R1ACAGCTC_R2TCGGTGA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19779_ModernCalculus_PerioDisease_13__R1TGACTGG_R2GTCCAAT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19780_ModernCalculus_PerioDisease_14__R1ACGGTCT_R2AGTTATC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19781_ModernCalculus_PerioDisease_15__R1CAATTGC_R2TAGGCCA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19782_ModernCalculus_PerioDisease_16__R1GCCAATG_R2AGATTCC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19783_ModernCalculus_PerioDisease_17__R1TGGCCAT_R2CTCAAGG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19784_ModernCalculus_PerioDisease_18__R1ATTGGCA_R2GAGCCTT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19785_ModernCalculus_PerioDisease_19__R1CGATGTA_R2TCTGGAA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19786_ModernCalculus_PerioDisease_20__R1CGTTAGA_R2AGTCACG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19787_ModernCalculus_PerioDisease_21__R1GTAACTC_R2CTAGCGT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19788_ModernCalculus_PerioDisease_22__R1TACCGAG_R2GACTGTA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19789_ModernCalculus_PerioDisease_23__R1TACGTTC_R2TCGATAC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19790_ModernCalculus_PerioDisease_24__R1ACGTAAG_R2CGCCATG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19791_ModernCalculus_PerioDisease_25__R1CGTACCT_R2GTGGCAT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19792_ModernCalculus_PerioDisease_26__R1GTCATAC_R2CTAGACA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19793_ModernCalculus_PerioDisease_27__R1GTACGGA_R2TATTGCA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19794_ModernCalculus_PerioDisease_28__R1CTACTCG_R2ACAATGC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19795_ModernCalculus_PerioDisease_29__R1TAGCACG_R2GACTCGC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19796_ModernCalculus_PerioDisease_30__R1ACTGCGT_R2TCGAGTG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19797_ModernCalculus_PerioDisease_31__R1GACGAGT_R2AGGCCTA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19798_ModernCalculus_PerioDisease_32__R1TCGTCTA_R2CTTGGAC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19799_ModernCalculus_PerioDisease_33__R1GGTATCG_R2AGTCTAT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19800_ModernCalculus_PerioDisease_34__R1AGTAGAC_R2GAATTCG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19801_ModernCalculus_PerioDisease_35__R1GGAGTAC_R2TCCAAGT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19802_ModernCalculus_PerioDisease_36__R1TTCTACG_R2CCTTACG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19804_EBC1_B1ModernCalculus_PerioDisease__R1TTACAGT_R2CCTGCAG.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19805_EBC2_B1ModernCalculus_PerioDisease__R1AACGCTA_R2GGATGCT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19834_EBC1_B2ModernCalculus_PerioDisease__R1CCGTGAC_R2TTCATGA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19835_EBC2_B2ModernCalculus_PerioDisease__R1AAGACGT_R2GGAACGT.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19837_EBC1_B3ModernCalculus_PerioDisease__R1CCTCGTA_R2TTCCGTA.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19838_EBC2_B3ModernCalculus_PerioDisease__R1ACTTGGC_R2AAGGTAC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19957_EBC1_Monongahela_Reextracted__R1GTCCAAT_R2CATAGGC.collapsed.rma6
2014nr-LCAdefault/Emily-Perio-Monongahela-2018/19959_EBC2_Monongahela_Reextracted__R1CCTGCAG_R2GCGTAGT.collapsed.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/13224_EBC_IndRev.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/13224_IndRev_Switzerland.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_13cyc_1to1000dil.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_13cyc_1to100dil.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_13cyc_1to10dil.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_13cyc_Normal.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_30cyc_1to1000dil.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_30cyc_1to100dil.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_30cyc_1to10dil.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/15524_AdaptorDimer_30cyc_Normal.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/16264_EBC_York_UK.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/17930p_Human_Gobero_tooth_PB.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/17931p_Human_Gobero_petrosal_PB.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/17937p_Human_Gobero_petrosal_PB.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/17939p_Human_Gobero_petrosal_PB.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/17950p_Human_Gobero_tooth_PB.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/17951p_Human_Gobero_petrosal_PB.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/18459_EBC_2_Greenland.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/18699_EBC_2_Hungary.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/19022_EBC_2_African_b1.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/19079_EBC2__African_2.rma6
2017GCS_LCAweighted80-supp0-1-ms-me0-01/1_DC/180510_INS_ESkLWeMAb_AdaptorDimer-AGFDecon-StMarys-TimeStorage-Resequence/19106_EBC2_Hungary_Lachish_PB.rma6
```
The above list has been truncated. Meganserver finds 4665 .rma6 files.

> What does http://10.102.151.16:8001/megan6server/about report?
```bash
MeganServer
Version: MEGAN Community Edition (version 6.20.11, built 2 Nov 2020)
Hostname: comp-lc-0016.acib.production.int.aci.ics.psu.edu
IP address: 10.102.151.16
Port: 8001
Known users: 2
Total requests: 2
Server started: Wed Nov 04 11:54:16 EST 2020
Help: http://10.102.151.16:8001/megan6server/help
```

> Are you running MEGAN on the same machine as MeganServer?

Yes, I'm running MEGAN on the same machine as Meganserver. I have the same issue if I run each on a different machine. I figured it may be easier to debug if I isolate on a single machine.

> If you run the MEGAN with command line option -d then you will see all the requests that MEGAN makes. To help me debug, could you please copy and paste some of them into your web-browser to see whether they work there.
Then we would know whether MEGAN is making the right requests, but is failing due to permissions or something.

I will report back with this information as soon as the updated version is released.

Thanks very much for your quick response and help with this.

**Comment by jpetucci** [comment]
Any request reported by MEGAN (when run with the -d option) that I randomly tested worked OK in the browser returning an integer value. 

I've attached the MEGAN debug output when I try connecting to an instance of meganserver which was launched using a large input directory, which contains over 4000 files here: [meganserver_largest_directory.txt](https://github.com/husonlab/megan-ce/files/5489913/meganserver_largest_directory.txt)

When I use a much smaller input directory to start meganserver (86 files), I'm able to connect to meganserver from MEGAN without issue after the database is rebuilt. See MEGAN debug output here: [meganserver_small_directory.txt](https://github.com/husonlab/megan-ce/files/5489914/meganserver_small_directory.txt)

I guess the above could indicate many different things. Is there a connection time out or a limit on the number of files meganserver can handle? If one of the files in the large directory is corrupted or otherwise incorrectly formatted, would that cause a problem?

**Comment by husonlab** [comment]
Thank you, that is helpful. I haven't tested with that large number (4000) of files yet... I will run tests on that number of files and will see what the problem is.

**Comment by hjruscheweyh** [comment]
Hi @jpetucci, Hi Daniel

MEGANServer also reports a log file that is accessible via the admin interface (`http://10.102.151.16:8001/megan6server/admin/getLog` or similar depending on the address of your server.). You will get the backend errors there. That will be more informative as the error happens there. The simplest assumption is that one of the `daa` files is corrupt.

Let me know if and how I could help.

Best,
Hans

**Comment by jpetucci** [comment]
This issue is resolved in the latest version of Megan6 (v6.20.13).
